### ISSUE TITLE
Custom Message When Exiting an App

### ISSUE THREAD

**Issue by Alek99** [reflex-dev/reflex#1261]
Currently we show something like a keyboard interrupt.

```Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.10/3.10.8/Frameworks/Python.framework/Versions/3.10/lib/python3.10/threading.py", line 1567, in _shutdown
    lock.acquire()
KeyboardInterrupt: 
```
 Maybe in the future we can show a custom message such as 
 
`Reflex app stopped at time :....`

**Comment by ItIsOHM** [comment]
Hey there @Alek99, I'd love to contribute as a beginner. Can you please assign this to me and also help me as to where I can start to tackle this issue from?

**Comment by Alek99** [comment]
Awesome just assigned it to you @ItIsOHM

**Comment by ItIsOHM** [comment]
> Awesome just assigned it to you @ItIsOHM

Thanks so much!! Can you please tell me where I can start to fix this issue? :D

**Comment by QuietPigeon2001** [comment]
Hi @Alek99, although this is not assigned to me, I seem to have found a solution for this. May I work on this as well?

**Comment by ItIsOHM** [comment]
> Hi @Alek99, although this is not assigned to me, I seem to have found a solution for this. May I work on this as well?

Hey @QuietPigeon2001, I'm not actively working on this issue anymore so you can get it assigned to yourself, good luck!

**Comment by QuietPigeon2001** [comment]
> > Hi @Alek99, although this is not assigned to me, I seem to have found a solution for this. May I work on this as well?
> 
> 
> 
> Hey @QuietPigeon2001, I'm not actively working on this issue anymore so you can get it assigned to yourself, good luck!

@ItIsOHM Gotcha, thank you!

**Comment by masenf** [comment]
fixed via #1345
